src/cron.cc

```diff
@@ -29,7 +29,7 @@ Status Cron::SetScheduleTime(const std::vector<std::string> &args) {
     if (!s.IsOK()) {
       return Status(Status::NotOK, "time expression format error : " + s.Msg());
     }
-    new_schedulers.push_back(st);
+    new_schedulers.emplace_back(std::move(st));
```

Comment:
i think it is not a good idea to call move, since `st` may be access again

Reply:
actually, I also did that in parsing requests, but i find c++ standard doesn't recommend that, i just think we don't call that frequently.

Reply:
And i find we don't define its `move constructor`, so this change doesn't take any improvements. And all members of schedule just are `int` type.

Reply:
Yep, at this point, std::move was not a recommendable operation for the user-defined `st` object. I would revoke it here.